### ISSUE TITLE
Decklink displaying ".join(';')}" at the end of each link

### ISSUE THREAD

**Issue by jimmyk3008** [cr-api/cr-api#244]
## Request Details:
```
GET
http://api.cr-api.com/player/20LQJGR9Q
```

## Response Details:

```
Code: 200

https://link.clashroyale.com/deck/en?deck=28000005,26000062,26000004,26000032,26000026,26000034,26000030,28000011).join(';')}

```

## Issue Report:
 1. Did you encounter this recently, or has this bug always been there? Recently
 2. Expected behaviour: exclude the join info and have ';' as the separator

**Comment by jimmyk3008** [comment]
The rarity is also not consistent with others in that all other cards have a capital as the first letter of Common, Rare, Epic, Legendary. It's now showing as 'epic'

**Comment by selfish** [comment]
Fixed in `v10.2.19`
